testing/uacme/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Eivind Uggedal <eu@eju.no>
+# Maintainer: Eivind Uggedal <eu@eju.no>
+pkgname=uacme
+pkgver=1.0.10
+pkgrel=0
+pkgdesc="Lightweight client for the RFC8555 ACMEv2 protocol"
+url="https://github.com/ndilieto/uacme"
+arch="all"
+license="GPL-3.0-or-later ISC MIT"
+makedepends="openssl-dev curl-dev asciidoc"
+subpackages="$pkgname-doc"
+source="uacme-$pkgver.tar.gz::https://github.com/ndilieto/uacme/archive/v$pkgver.tar.gz"
+builddir="$srcdir/uacme-$pkgver"
+
+build() {
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--disable-maintainer-mode \
+		--with-openssl
+	make
+}
+
+check() {
+	2>&1 ./uacme -V | grep "$pkgver"
```

Comment:
Running just `$program -v` is not sufficient for a check. It usually only just a tiny portion of the code, giving basically no coverage. In that case, we prefer the package to be marked as not having a test suite (`options="!check"`).

Reply:
While having an actual test suite is best, I don’t agree that it’s equivalent to !check for a compiled program with dynamic libraries, but I will remove it.